VagrantProvision.sh

```diff
@@ -227,6 +239,7 @@ if ! ogrinfo --formats | grep --quiet FileGDB; then
     echo "### Building GDAL w/ FileGDB..."
     export PATH=/usr/local/lib:/usr/local/bin:$PATH
     cd gdal-1.10.1
+    touch config.rpath
```

Comment:
Does this get rid of a clock skew warning or something?
Or did the file not exist and needs to?